db.go

```diff
@@ -206,6 +212,17 @@ func makeUpdatePlan(m *Model) updatePlan {
 		}
 		setColumns = append(setColumns, col.Name)
 		p.setColumns = append(p.setColumns, m.Table.C(col.Name))
+		var setter func(reflect.Value, int) interface{}
```

Comment:
It would shorten the code only very slightly, and introduce one more name. Net-net, I don't see it as clearer than this.